### ISSUE TITLE
Certificate shows as expired after updating the certificate

### ISSUE THREAD

**Issue by nishantkagrawal** [blupig/certificate-info#20]
I updated my Certificate which was expiring soon.

It still shows as expired after the certificate is updated.

![image](https://user-images.githubusercontent.com/8187427/79562247-5e2c5680-80c8-11ea-9d59-0ffe7a6df0d5.png)
![image](https://user-images.githubusercontent.com/8187427/79562438-ba8f7600-80c8-11ea-9ebf-92f4612e3898.png)

@kkostrzewa

**Comment by kkostrzewa** [comment]
@nishantkagrawal there is server side caching to lessen the load. 

every 8 hours is a cache refresh
https://github.com/blupig/certificate-info/blob/master/server/main.go#L336

**Comment by yunzhu-li** [comment]
I think the data is also [cached on client side](https://github.com/blupig/certificate-info/blob/master/certificate-info/background.js#L117) until the extension is reloaded or browser restarted. Should probably add refresh on client side as well.

**Comment by kkostrzewa** [comment]
could be handled by a cache busting UI which passed a new param to the BE as well.

**Comment by yunzhu-li** [comment]
Right. Ideally there's a UI to configure the plugin, but I currently don't have the bandwidth for it. I'll try to add a simple periodic refresh for now so at least the client data can be updated.

**Comment by flunda** [comment]
thanks for adding the feature for displaying the certificate's expiration date!!
